solar-system/src/org/gearvrf/solarsystem/SolarViewManager.java

```diff
@@ -181,17 +190,32 @@ private void counterClockwise(GVRSceneObject object, float duration) {
                 0.0f, 0.0f, 0.0f));
     }
 
-    private void clockwise(GVRSceneObject object, float duration) {
-        setup(new GVRRotationByAxisWithPivotAnimation( //
-                object, duration, -360.0f, //
-                0.0f, 1.0f, 0.0f, //
-                0.0f, 0.0f, 0.0f));
-    }
-
     private void clockwise(GVRTransform transform, float duration) {
         setup(new GVRRotationByAxisWithPivotAnimation( //
                 transform, duration, -360.0f, //
                 0.0f, 1.0f, 0.0f, //
                 0.0f, 0.0f, 0.0f));
     }
+
+    public void applySceneLightMap(GVRScene scene, GVRContext gvrContext) {
```

Comment:
Are you planning to do that as part of the existing pull request or maybe later? Thanks.